scripts/servers_test.py

```diff
@@ -631,8 +631,11 @@ def test_managed_web_browser_on_virtualbox_os(self) -> None:
         self.exit_stack.enter_context(self.swap_to_always_return(
             os, 'listdir', value=['VBOX-123']))
```

Comment:
Do we still need to mock the VBOX?

Reply:
You are right. This is not needed now. I will remove it.